node/light.go

```diff
@@ -92,6 +96,9 @@ func openDatastore(conf config.NodeConfig, logger log.Logger) (ds.TxnDatastore,
 	return store.NewDefaultKVStore(conf.RootDir, conf.DBPath, "rollkit-light")
 }
 
+// OnStart starts the P2P and HeaderSync services
+//
+// TODO: should this just be Start?
```

Comment:
the base service just has start and stop. Also in the absence of a Start method, OnStart seems out of place, as it would imply that there is a start function that this triggers off.

Reply:
The `Start` method in `BaseService` calls the `OnStart` method of its `impl`

Reply:
```suggestion
```